### PR TITLE
Update CONDUCT.md

### PR DESCRIPTION
**Pull request by stevenleung2018**

### CODE REVIEW

**Comment by stevenleung2018** [comment]
Since the team contract has contents pertaining how the whole team work together, I am assigning the whole team as reviewers.  Please review and approve.  And the last person approve should merge the PR.

File: CONDUCT.md [hunk 1]
```diff
@@ -24,6 +24,13 @@ Examples of unacceptable behavior by participants include:
 
 ## Our Responsibilities
 
+Our team members are committed to ensuring an inclusive environment and addressing any behaviour that does not align with the Code of Conduct. Our team members currently include:
+
+* LG Nguyen
```
Comment (gn385x) [inline CONDUCT.md:29]: hi Steven, could you change my name here to Linh Giang Nguyen (that's how it's officially recorded hehe)? thanks!

File: team-contract.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Team Contract
+
+## Team Members
+* LG Nguyen
```
Comment (gn385x) [inline team-contract.md:4]: here as well: Linh Giang Nguyen

**Review by gn385x — CHANGES_REQUESTED** [review]
besides changing my name, i approve of all changes!

File: CONDUCT.md (hunk 1 above)
Comment (stevenleung2018) [reply CONDUCT.md]: Thanks for pointing out.  Changed and committed.

**Comment by stevenleung2018** [comment]
FYI, I have also committed changes to the CONTRIBUTING.md for changing the name of the project.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,10 @@
-# doggodash
+# doggoash
```
Comment (squisty) [inline README.md:1]: is this misspelled now? Could be an error on my end.

**Pull request merged**